cmd/mcp-server.py

```
@@ -0,0 +1,231 @@
+    "release-2.15",
+    "release-2.16",
+]
+DEFAULT_RELEASE = KNOWN_RELEASES[-1]
+
+RAW_BASE = "https://raw.githubusercontent.com/stolostron/api-documentation/{release}/api-docs/ai"
+
+# ── HTTP helpers ─────────────────────────────────────────────────────────────
+
+def _fetch_json(url: str) -> dict:
+    """Fetch and parse JSON from a URL. Raises urllib.error.HTTPError on failure."""
+    with urllib.request.urlopen(url, timeout=15) as resp:
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)